app/presenters/notices/setup/notifications.presenter.js

```diff
@@ -177,9 +177,10 @@ function _letterTemplate(contactType, journey) {
 
 function _returnsPeriod(returnsPeriod) {
   return {
-    periodEndDate: formatLongDate(new Date(returnsPeriod.endDate)),
-    periodStartDate: formatLongDate(new Date(returnsPeriod.startDate)),
-    returnDueDate: formatLongDate(new Date(returnsPeriod.dueDate))
+    // periodEndDate: formatLongDate(new Date(returnsPeriod.endDate)),
+    // periodStartDate: formatLongDate(new Date(returnsPeriod.startDate)),
+    // returnDueDate: formatLongDate(new Date(returnsPeriod.dueDate))
+    // flow_or_level, monitoring_station_name, licence_ref, issuer_email_address, source, threshold_value, threshold_unit, condition_text
```

Comment:
This is the most required template structure. 

What is condition text ?

Reply:
To get the thresholds we need to use the same logic we had previously to determine the thresholds to use. Maybe lift the logic out of the presenter into a determine service.